app/src/main/java/com/eduard/caloriecounter/presentation/activity/NotesListActivity.java

```diff
@@ -0,0 +1,107 @@
+package com.eduard.caloriecounter.presentation.activity;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.TextView;
+
+import com.eduard.caloriecounter.R;
+import com.eduard.caloriecounter.data.repository.NoteRepository;
+import com.eduard.caloriecounter.presentation.adapter.NotesListAdapter;
+import com.eduard.caloriecounter.presentation.constants.AppConstants;
+import com.eduard.caloriecounter.presentation.fragments.CollectionFragment;
+import com.eduard.caloriecounter.presentation.model.Note;
+import com.eduard.caloriecounter.presentation.utils.NavigatorUtils;
+import com.eduard.caloriecounter.presentation.utils.RecyclerItemClickListener;
+import com.google.android.material.floatingactionbutton.FloatingActionButton;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.RecyclerView;
+import androidx.recyclerview.widget.StaggeredGridLayoutManager;
+
+public class NotesListActivity extends AppCompatActivity implements View.OnClickListener, RecyclerItemClickListener.OnRecyclerViewItemClickListener, AppConstants {
```

Comment:
Remove AppConstants. Constants are static variables that you should **import**, and **not implement** like interfaces.

Reply:
ready